pymysql/converters.py

```diff
@@ -89,9 +89,11 @@ def escape_time(obj):
     return escape_str(s)
 
 def escape_datetime(obj):
+    obj = obj.replace(tzinfo=None)
```

Comment:
For MySQL, it's true that there is no timezone support in datetime formats but it handles the timezone as follows:
- `TIMESTAMP` gets stored as UTC and will be converted from/to current `session.time_zone` on INSERT/UPDATE/SELECT
- `DATETIME` doesn't handle time zones at all

Normally you setup the timezone in MySQL on the server once and never change that (hopefully UTC) but you can change the time zone on runtime by `SET @@session.time_zone = 'Europe/Berlin'`.

I think to make it right you should convert the `datetime` object to the time zone provided by `SELECT IF(@@session.time_zone="SYSTEM",@@global.time_zone,@@session.time_zone)` as long as the object has a time zone.

I know this would be slow on doing it each time escape gets called but it could be cached in the connection.

Thoughts ?